docs/usage/configuration-options.md

```diff
@@ -681,6 +681,9 @@ Example:
 
 ## gitLabAutomerge
 
+If you enabled automerge in the Renovate config, you can speed up the automerge process, by using GitLab's own automerge function.
+Attention: This is a top-level key.
+
```

Comment:
```suggestion
```

I think thats's not true, you can also use `gitLabAutomerge` in a package rule.

Reply:
https://github.com/renovatebot/renovate/blob/34ef699b64b5de1c6f9fe70eafbb99422b839081/lib/workers/pr/index.ts#L115-L128